test/liblll/EndToEndTest.cpp

```diff
@@ -165,6 +165,31 @@ BOOST_AUTO_TEST_CASE(conditional_seq)
 	BOOST_CHECK(callFallback() == toBigEndian(u256(1)));
 }
 
+BOOST_AUTO_TEST_CASE(conditional_nested)
```

Comment:
I'll separate it out into a seoarate PR tomorrow. I have converse case already, so will add that as well.